app/templates/suppliers/duns_number.html

```diff
@@ -18,6 +18,17 @@
 
 {% block main_content %}
 
+{% if form.duns_number.errors[0] == 'DUNS number already used' %}
+<div class="validation-masthead" role="group" aria-labelledby="validation-masthead-heading">
+    <h1 class="validation-masthead-heading" id="validation-masthead-heading">
+        A supplier account already exists with that DUNS number
```

Comment:
The short version that gets passed in is shown unedited in-line next to the input box.  One of the places where the message is displayed will need to rewrite it, and I think it's nicer to keep the thing we test as the short one.